moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -327,39 +299,37 @@ const ProjectFundingTable = () => {
     }, {});
   };
 
-  /**
-   * Component for dropdown select using a lookup table as options
-   * @param {*} props
-   * @returns {React component}
-   */
-  const LookupSelectComponent = (props) => (
-    <Select
-      style={{ minWidth: "8em" }}
+/**
+ * Component for autocomplete using a lookup table as options
+ * @param {*} props
+ * @returns {React component}
+ */
+  const LookupAutocompleteComponent = (props) => (
+    <Autocomplete
+      style={{ minWidth: "200px" }}
+      value={
+        // if we are editing, the autocomplete has the value provided by the material table, which is the record id
+        // need to get its corresponding text value
+        props.value
+          ? getLookupValueByID(props.lookupTableName, props.name, props.value)
```

Comment:
this is the reason I kept it in here, its using getLookupValueByID, though now that I think about it, I could also put that in a separate file and just pass the data into it

Reply:
ah yes, thanks. 👍 🚢